index.html

```diff
@@ -298,62 +297,62 @@
 
 
 	<section class="slide gap">
-		<h2>Параллельная работа</h2>
+		<h2>Concurrent working</h2>
 	</section>
 
 	<section class="slide">
-		<h2>Метрики производительности</h2>
+		<h2>Performance metrics</h2>
 		<ul>
-			<li><b>Реальное время выполнения одной операций</b><br>
-				Важно на десктопе.
+			<li><b>The real time for one operation execution</b><br>
+				Important on desktops.
 			</li>
-			<li><b>Пропускная способность потока операций</b><br>
-				Становится более важным на сервере.
+			<li><b>Throughput of operations flow</b><br>
+				Becomes more important on servers.
 			</li>
 		</ul>
 	</section>
 
 	<section class="slide">
-		<h2>Способы параллельной работы</h2>
+		<h2>Concurrent working levels</h2>
 		<ol>
-			<li class="non-zero" data-counter="1">На уровне приложения</li>
+			<li class="non-zero" data-counter="1">Application level</li>
 		</ol>
 		<p>
-			Реальное время выполнения не меняется.<br>
-			Пропускная способность растет пропорционально количеству ядер.
+			The real time doesn't change.<br>
+			Throughput grows in proportion to the number of cores.
```

Comment:
Actual execution time does not change
... `grows | increases` in proportion ...